src/main/webapp/partials/instructorHelpSessions.jsp

```diff
@@ -566,10 +566,7 @@
             </form>
             <br>
             <br>
-          </div>
-
         </div>
-
```

Comment:
Revert this unnecessary removal of a blank line.

Reply:
Done